docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -666,7 +666,7 @@ The entries are relative to the root of the generated Uber-Jar. You can specify
 By default, Quarkus will not discover CDI beans inside another module.
 
 The best way to enable CDI bean discovery for a module in a multi-module project would be to include a `META-INF/beans.xml` file,
-unless it is the main application module already configured with the quarkus-maven-plugin, in which case it will be indexed automatically.
+unless it is the main application module already configured with the quarkus-gradle-plugin, in which case it will be indexed automatically.
```

Comment:
Thanks, it's a good catch. I think I would go with:
```suggestion
unless it is the main application module already configured with the `io.quarkus` Gradle plugin, in which case it will be indexed automatically.
```

Will adjust and merge.